.azure/pipelines/ci.yml

```diff
@@ -184,10 +184,9 @@ stages:
                 -noBuildRepoTasks
                 -pack
                 -noBuildDeps
-                -noBuildNative
```

Comment:
If something is changing here, may need to change src\SiteExtensions\build.cmd instead of doing this for _all_ of the projects built in that script i.e. only change the command building src/SiteExtensions/Runtime/Microsoft.AspNetCore.Runtime.SiteExtension.pkgproj

Reply:
I still think we need to remove this from all places. I think NoBuildNative takes precedence over BuildNative.

Reply:
Also, isn't this the only location?

Reply:
`-noBuildNative` does take precedence over `-buildNative` but that should be fine in the YAML because we only use `-noBuildNative` in build steps where something earlier has already built the usual native projects.

Reply:
Okay, is there anything I need to change here then?

Reply:
Shouldn't be but it may be necessary to make sure the specific native projects Microsoft.AspNetCore.Runtime.SiteExtension.pkgproj references are built. Might need to force build of Microsoft.AspNetCore.Runtime.SiteExtension.pkgproj to use `msbuild` using `-buildNative` in the script the first time it builds that project (second time might be fine if native projects build both bitnesses the first time).

Reply:
From what I can tell, this works now.

Reply:
From what I see in the logs, removing this means projects like CommonLibTests.vcxproj are built repeatedly even though the x64 build step already built them. That's just wasted CPU cycles.

I double-checked and all `*.vcxproj` projects should be built in the x64 build step. See https://github.com/dotnet/aspnetcore/blob/918172f6deb19d78ac28886232d4b4a893f72025/eng/Build.props#L112-L119

That said, may need code like that at https://github.com/dotnet/aspnetcore/blob/918172f6deb19d78ac28886232d4b4a893f72025/src/Framework/App.Runtime/src/Microsoft.AspNetCore.App.Runtime.csproj#L139-L142 to get the output of the earlier built